seed-repo

```diff
@@ -75,9 +75,6 @@ MILESTONES.each do |m|
   end
 end
 
-# User must be an owner of an org to add the repository to teams
-puts "Attempting to give teams admin access to the repository."
-puts "This will fail if you aren't an owner on the org."
```

Comment:
It's not the case that users must be an owner of the org - I think you were the person who disproved it, right? You were able to add the teams to the repo just fine.

I think the creator of the repo has the ability to add teams.